### PR TITLE
improvement: Admin and subscriber notifications

### PR DESCRIPTION
**Pull request by skanderm**
- For #111

### CODE REVIEW

File: server/lib/orcasite/notifications/resources/notification.ex [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    notifiers: [Orcasite.Notifications.Notifiers.NotifySubscriptions]
+
+  alias Orcasite.Notifications.{Event, SubscriptionNotification, Subscription}
+
+  resource do
+    description """
+    A notification object. Once created, this will be sent to any number of subscribers (api clients,
+    newsletters, individuals, admins, etc) through their subscribed channels for the given event type
+    (confirmed candidate created, detection verified, etc).
+    """
```
Comment (paulcretu) [inline server/lib/orcasite/notifications/resources/notification.ex:14]: I find the distinction here in these descriptions between `notification` and `subscription_notification` slightly confusing. How I understand it (mostly based on previous discussions): there's one `notification` object per event, and then a `subscription_notification` is created for each subscriber. So `subscription_notification` represents the actual individual notification instance that goes out to a specific subscriber, while `notification` is like a blueprint / template. Do I have that right?

I think it's inherently confusing so I want to make sure we document it properly

**Review by paulcretu — APPROVED** [review]

File: server/lib/orcasite/notifications/resources/notification.ex (hunk 1 above)
Comment (skanderm) [reply server/lib/orcasite/notifications/resources/notification.ex:14]: Yep that's exactly it. `SubscriptionNotification` is the join table between a `Notification` (send this event to everyone subscribed) and `Subscription` (subscribe to an event type with a particular channel)

File: server/lib/orcasite/notifications/email.ex [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    }
+
+    ~H"""
+    <div>
+      <p>
+        Don't miss the concert!
+      </p>
+
+      <p>
+        Listen to orcase here: <a href={"https://live.orcasound.net/#{@node}"}>https://live.orcasound.net/<%= @node %></a>
```
[the remark this answers is not in the document]
Comment (skanderm) [reply server/lib/orcasite/notifications/email.ex:62]: Oh yeah, I can get that fixed.

File: server/Procfile [hunk 3]
```diff
@@ -1 +1 @@
-web: MIX_ENV=prod mix phx.server
\ No newline at end of file
+web: MIX_ENV=prod URLS="$URLS //$HEROKU_APP_NAME.herokuapp.com" mix phx.server
```
Comment (paulcretu) [inline server/Procfile:1]: You should be able to remove this now that I merged my PR with the script

**Review by paulcretu — DISMISSED** [review]

**Pull request merged**